config/locales/doorkeeper.en.yml

```diff
@@ -0,0 +1,124 @@
+en:
```

Comment:
If it no longer translates when this is gone, then just keep it ... just trying to avoid copy-paste and upgrade pains